src/gwt/src/org/rstudio/studio/client/workbench/views/environment/view/MemoryUsageSummaryDialog.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * MemoryUsageSummaryDialog.java
+ *
+ * Copyright (C) 2021 by RStudio, PBC
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+package org.rstudio.studio.client.workbench.views.environment.view;
+
+import com.google.gwt.aria.client.Roles;
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.user.client.ui.Widget;
+import org.rstudio.core.client.widget.ModalDialogBase;
+import org.rstudio.core.client.widget.ThemedButton;
+import org.rstudio.studio.client.workbench.views.environment.model.MemoryUsageReport;
+
+public class MemoryUsageSummaryDialog extends ModalDialogBase
+{
+   public MemoryUsageSummaryDialog(MemoryUsageReport report)
+   {
+      super(Roles.getDialogRole());
+      summary_ = new MemoryUsageSummary(report);
+      setText("Memory Usage Report");
```

Comment:
Nice to have for accessibility: include the value in the title, e.g.: "Memory Usage Report (39% in use)". 

That gives screen-reader user immediate value out of this dialog the moment they launch it, otherwise they have to navigate the internals of the dialog to discover that summary value.

Reply:
Good idea!